independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/model/AppArtifact.java

```diff
@@ -12,7 +10,9 @@
  */
 public class AppArtifact extends AppArtifactCoords implements Serializable {
 
-    protected transient Path path;
+    private static final long serialVersionUID = 1L;
```

Comment:
Do we actually want this? If we forget to update it we could have weird errors when upgrading.